sonar/theme/views.py

```diff
@@ -147,8 +147,8 @@ def logged_user():
                 'list': DepositPermission.list(user)
             },
             'projects': {
-                'add': ProjectPermission.create(user),
-                'list': ProjectPermission.list(user)
+                'add': ProjectPermissionPolicy('create').can(),
```

Comment:
Probably we can use in a near future, the REST API of invenio-accounts.

Reply:
OK